app/views/_vote_buttons.erb

```diff
@@ -0,0 +1,11 @@
+<div id="vote-wrapper">
+  <form id="upvote" method="post" action='/suggestions/<%= @suggestion.id %>/vote'>
+    <input type="hidden" name="suggestion_id" value="<%= params[:id] %>">
```

Comment:
Don't pull from params in the views, pull from the instance variables passed to the view.